src/main/java/me/coley/recaf/util/IllegalBytecodePatcherUtil.java

```diff
@@ -1,6 +1,6 @@
 package me.coley.recaf.util;
 
-import me.coley.cafedude.ClassFile;
+import me.coley.cafedude.classfile.ClassFile;
```

Comment:
Part of the 1.8 to 1.9 transition requires adding `new IllegalStrippingTransformer(cf).transform();` to retain the same behavior of stripping invalid attributes.